content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -79,51 +79,51 @@ The pod network plugin you use (see below) may also require certain ports to be
 open. Since this differs with each pod network plugin, please see the
 documentation for the plugins about what port(s) those need.
 
-## Installing runtime {#installing-runtime}
+## Installing a container runtime {#installing-runtime}
 
 To run containers in Pods, Kubernetes uses a
 {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}.
 
-{{< tabs name="container_runtime" >}}
-{{% tab name="Linux nodes" %}}
-
 By default, Kubernetes uses the
 {{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI)
 to interface with your chosen container runtime.
 
 If you don't specify a runtime, kubeadm automatically tries to detect an installed
-container runtime by scanning through a list of well known Unix domain sockets.
-The following table lists container runtimes and their associated socket paths:
-
-{{< table caption = "Container runtimes and their socket paths" >}}
-| Runtime    | Path to Unix domain socket        |
-|------------|-----------------------------------|
-| Docker     | `/var/run/dockershim.sock`        |
-| containerd | `/run/containerd/containerd.sock` |
-| CRI-O      | `/var/run/crio/crio.sock`         |
-{{< /table >}}
-
-<br />
-If both Docker and containerd are detected, Docker takes precedence. This is
-needed because Docker 18.09 ships with containerd and both are detectable even if you only
-installed Docker.
-If any other two or more runtimes are detected, kubeadm exits with an error.
+container runtime by scanning through a list of known endpoints.
 
-The kubelet integrates with Docker through the built-in `dockershim` CRI implementation.
+If multiple or no container runtimes are detected kubeadm will throw an error
+and will request that you specify which one you want to use.
 
 See [container runtimes](/docs/setup/production-environment/container-runtimes/)
 for more information.
+
+This tables below include the known endpoints for supported operating systems:
+
+{{< tabs name="container_runtime" >}}
+{{% tab name="Linux" %}}
+
+{{< table >}}
+| Runtime      | Path to Unix domain socket                   |
+|--------------|----------------------------------------------|
+| containerd   | `unix:///var/run/containerd/containerd.sock` |
+| CRI-O        | `unix:///var/run/crio/crio.sock`             |
+| Docker       | `unix:///var/run/cri-dockerd.sock`           |
+{{< /table >}}
+
 {{% /tab %}}
-{{% tab name="other operating systems" %}}
-By default, kubeadm uses {{< glossary_tooltip term_id="docker" >}} as the container runtime.
-The kubelet integrates with Docker through the built-in `dockershim` CRI implementation.
 
-See [container runtimes](/docs/setup/production-environment/container-runtimes/)
-for more information.
+{{% tab name="Windows" %}}
+
+{{< table >}}
+| Runtime      | Path to Windows named pipe                   |
+|--------------|----------------------------------------------|
+| containerd   | `npipe:////./pipe/containerd-containerd`     |
```

Comment:
This looks like a type, "containerd-containerd" ?

At least when compared to https://github.com/kubernetes-sigs/cri-tools/blob/v1.23.0/docs/crictl.md#usage

Reply:
It is how it was originally contributed in kubeadm by someone from sig windows :
https://github.com/kubernetes/kubernetes/commit/310bafe658bce686e7286f9b5a0d7e4336581ecd#diff-f6120441e4bf907c82c49dd0f3a9ef25f1b6ce6ad9f4fb41dda0ef2e3bf9aae9R27

@marosset is the common named pipe for containerd accurate here?

Reply:
These pages have the containerd-containerd npipe:
https://www.jamessturtevant.com/posts/Windows-Containers-on-Windows-10-without-Docker-using-Containerd/

https://projectcalico.docs.tigera.io/getting-started/windows-calico/kubernetes/standard

Reply:
Could be a bug in crictl 1.23.0 docs then

Reply:
That looks like the correct default endpoint to me.
Running `containerd.exe config default` gives

>>>
[grpc]
  address = "\\\\.\\pipe\\containerd-containerd"
  gid = 0
  max_recv_message_size = 16777216
  max_send_message_size = 16777216
  tcp_address = ""
  tcp_tls_cert = ""
  tcp_tls_key = ""
  uid = 0
>>>

And then this entry fixes up the slashes for Windows and prefixes the path with `npipe` which is OK

Reply:
thanks for confirming!